libs/payments/ui/src/lib/client/components/Header/index.tsx

```diff
@@ -0,0 +1,372 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use client';
+
+import Image from 'next/image';
+import helpIcon from '@fxa/shared/assets/images/help.svg';
+import mozillaIcon from '@fxa/shared/assets/images/moz-logo-bw-rgb.svg';
+import bentoIcon from '@fxa/shared/assets/images/bento.svg';
+import defaultAvatarIcon from '@fxa/shared/assets/images/avatar-default.svg';
+import closeIcon from '@fxa/shared/assets/images/close.svg';
+import desktopIcon from '@fxa/shared/assets/images/desktop.svg';
+import mobileIcon from '@fxa/shared/assets/images/mobile.svg';
+import monitorIcon from '@fxa/shared/assets/images/monitor.svg';
+import relayIcon from '@fxa/shared/assets/images/relay.svg';
+import vpnIcon from '@fxa/shared/assets/images/vpn-logo.svg';
+import pocketIcon from '@fxa/shared/assets/images/pocket.svg';
+import signOutIcon from '@fxa/shared/assets/images/sign-out.svg';
+import { LinkExternal } from '@fxa/shared/react';
+import { Localized, useLocalization } from '@fluent/react';
+import { useState } from 'react';
+import { useClickOutsideEffect } from '../../hooks/useClickOutsideEffect';
+import { useEscKeydownEffect } from '../../hooks/useEscKeydownEffect';
+import { User } from 'next-auth';
+import { constructHrefWithUtm } from '../../../utils/constructHrefWithUtm';
+
+export const Header = ({
+  authContext,
```

Comment:
nit. Could this be renamed to remove `Context` to avoid mixing it up with React Context?